generators/client/templates/angular/src/main/webapp/app/account/password-reset/finish/password-reset-finish.component.ts.ejs

```diff
@@ -28,28 +30,32 @@ import { PasswordResetFinishService } from './password-reset-finish.service';
     templateUrl: './password-reset-finish.component.html'
 })
 export class PasswordResetFinishComponent implements OnInit, AfterViewInit {
-    confirmPassword: string;
     doNotMatch: string;
     error: string;
     keyMissing: boolean;
-    resetAccount: any;
     success: string;
     modalRef: NgbModalRef;
     key: string;
 
+    passwordForm = this.fb.group({
```

Comment:
an alternative could be to move form model outside in separate model class and do mapping between dto model and form model using services

Reply:
I wanted to stay coherent for form interaction between client and entity classes.
Generating a new model for form is a much time consuming and don't add much value (at least on my side, where forms are 3 to 5 attributes): may be doing this step by step if it is asked and voted (sorry in advance for those who will migrate)?